### PR TITLE
Change to LocalizeDynamicMixin

### PR DESCRIPTION
**Pull request by lchan131**

### CODE REVIEW

File: media-player.js [hunk 1]
```diff
@@ -20,8 +20,8 @@ import { FocusVisiblePolyfillMixin } from '@brightspace-ui/core/mixins/focus-vis
 import fullscreenApi from './src/fullscreen-api.js';
 import Fuse from 'fuse.js';
 import { ifDefined } from 'lit-html/directives/if-defined.js';
-import { InternalLocalizeMixin } from './src/mixins/internal-localize-mixin.js';
```
Comment (rcausey) [inline media-player.js:23]: do we still need this mixin?

File: media-player.js [hunk 2]
```diff
@@ -638,6 +638,14 @@ class MediaPlayer extends FocusVisiblePolyfillMixin(InternalLocalizeMixin(RtlMix
 	get isIOSVideo() {
 		return IS_IOS && this.mediaType === SOURCE_TYPES.video;
 	}
+	static get localizeConfig() {
```
Comment (qwangd2l) [inline media-player.js:641]: I'd prefer if we moved the static methods together if we're defining this within the component file.
The other option would be to refactor this into a mixin and use it like the old `InternalLocalizeMixin`.

File: media-player.js (hunk 1 above)
Comment (lchan131) [reply media-player.js:23]: No. It has been removed.

File: media-player.js (hunk 2 above)
Comment (rcausey) [reply media-player.js:641]: Yeah good point. I think keeping it as a mixin is a good idea rather than implementing directly in this file.

Comment (rcausey) [reply media-player.js:641]: See: https://github.com/Brightspace/content-components/blob/5ad01f515cd6dd5446b868739a0f92f8e03de535/core/d2l-content-renderer/src/mixins/internal-localize-mixin.js

[the remark this answers is not in the document]
Comment (lchan131) [reply media-player.js]: Lint rule added

Comment (lchan131) [reply media-player.js]: Moved to its own class

**Comment by qwangd2l** [comment]
Looks good to me, but you might want to consider removing the `internal-localize-mixin.js` file since it's no longer used (it's what Rhys commented about earlier).

File: media-player.js (hunk 1 above)
Comment (lchan131) [reply media-player.js:23]: That is strange. I am seeing it is highlighted red, so I'm positive it is removed. Try refreshing.

Comment (qwangd2l) [reply media-player.js:23]: The file, or the import statement? I think we're talking about the removal of the file.

**Review by rcausey — APPROVED** [review]
Looks good. Make sure to bump the version on merge.

**Review by qwangd2l — APPROVED** [review]

**Pull request merged**